### DISCUSSION TITLE
Add tab management.

### DISCUSSION THREAD

**Discussion by gauravsaini02** [deepgram/community#828] [General help] [answered]
I want to transcribe in multiple tabs at once. One tab should be isolated from another one. If transcription is going on in one tab, I should be able to go to another tab and turn on transcription for that particular tab.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by SandraRodgers** [comment]
Are you referring to browser tabs? Can you provide more information about the STT request? Would this be prerecorded or streaming?

**Comment by gauravsaini02** [comment]
Yes I am referring to chrome browser tabs. If I want to transcribe 2
youtube videos simultaneously.

**Reply by SandraRodgers** [reply, marked as answer]
It would be two isolated requests because you open up the new tab, and it runs the code for that distinct session. 

Are you running into some problem when you try to do this?
